htsget-search/src/bam_search.rs

```diff
@@ -598,7 +598,10 @@ pub(crate) mod tests {
   #[tokio::test]
   async fn search_all_c4gh() {
     with_local_storage_c4gh(|storage| async move {
-      let storage = C4GHStorage::new(get_decryption_keys(), Arc::try_unwrap(storage).unwrap());
+      let storage = C4GHStorage::new(
+        get_decryption_keys().await,
+        Arc::try_unwrap(storage).unwrap(),
```

Comment:
Is this Arc and so many unwraps necessary here?

Reply:
All gone now!